src/hooks/useDataSource.ts

```diff
@@ -124,7 +124,13 @@ async function importBook(storage: BookStorage | null, rawData: Uint8Array) {
   const rawBook = tempDataSource.sqljsManager.exportDatabase();
 
   await DATASOURCE.sqljsManager.loadDatabase(rawBook);
-  mutate((key: string) => key.startsWith('/api'));
+  mutate((key: string) => {
+    let k = key;
+    if (Array.isArray(key)) {
+      [k] = key;
+    }
+    return k.startsWith('/api');
+  }, undefined);
```

Comment:
set to `undefined` so instead of having stale data we have no data. This is good because otherwise it causes having inconsistent accounts vs prices